gtk/src/toga_gtk/widgets/slider.py

```diff
@@ -44,6 +41,22 @@ def create(self):
         # Dummy values used during initialization.
         self.tick_count = None
 
+    def gtk_on_change(self, widget):
+        if self.interface.on_change:
+            self.interface.on_change(widget)
```

Comment:
I'm not sure if I understand you correctly or not, but as I understand there is no need to check if `on_change` is none or not because we have `wrapped_handler` that handle this condition and deal with it in appropriate way. Is this correct? Is this only for `on_change` or for others like `on_press` and `on_release`? Is this for all widget or only for `slider`?

Reply:
It's true for *every* handler, on every widget. `wrapped_handler(self, None)` is a valid call, returning a no-op handler.

It also means you don't need to pass the widget that is generating the signal as an argument - the widget is bound as part of the wrapping process. We've been trying to replace `on_change(widget)` with `on_change(None)` to make this obvious; once the audit is complete, we should be able to update all the event handlers to remove the first argument.